README.md

```diff
@@ -1,5 +1,137 @@
 
-This project contains the runtime for the Kata Containers project.
+This repository contains the runtime for the
+[Kata Containers](https://github.com/kata-containers) project.
+
+For details of the other Kata Containers repositories, see the
+[repository summary](https://github.com/kata-containers/kata-containers).
+
+* [Introduction](#introduction)
+* [License](#license)
+* [Platform support](#platform-support)
+    * [Hardware requirements](#hardware-requirements)
+* [Quick start for developers](#quick-start-for-developers)
+* [Configuration](#configuration)
+* [Logging](#logging)
+* [Debugging](#debugging)
+* [Community](#community)
+
+## Introduction
+
+`kata-runtime`, henceforth known simply as "the runtime", is the
+CLI (command-line interface) part of the Kata Containers runtime component. It leverages the
+[virtcontainers](https://github.com/kata-containers/runtime/tree/master/virtcontainers)
+package to provide a high-performance standards-compliant runtime that creates
+hardware-virtualized containers.
+
+The runtime is both
+[OCI](https://github.com/opencontainers/runtime-spec)-compatible and
+[CRI-O](https://github.com/kubernetes-incubator/cri-o)-compatible, allowing it
+to work seamlessly with both Docker and Kubernetes respectively.
+
+## License
+
+The code is licensed under an Apache 2.0 license.
+
+See [the license file](LICENSE) for further details.
+
+## Platform support
+
+Kata Containers currently works on systems supporting the following
+technologies:
+
+- [Intel](https://www.intel.com)'s VT-x technology.
+- [ARM](https://www.arm.com)'s Hyp mode (virtualization extension).
+
+### Hardware requirements
+
+The runtime has a built-in command to determine if your host system is capable
+of running a Kata Container:
+
+```bash
+$ kata-runtime kata-check
+```
+
+> **Note:**
+>
+> If you run the command above as the `root` user, further checks will be
+> performed (e.g. it will check if another incompatible hypervisor is running):
+>
+> ```bash
+> $ sudo kata-runtime kata-check
+> ```
+
+## Quick start for developers
+
+See the
+[developer guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md).
+
+## Configuration
+
+The runtime uses a TOML format configuration file called `configuration.toml`.
+The file contains comments explaining all options.
+
+> **Note:**
+>
+> The initial values in the configuration file provide a good default configuration.
+> You may only need to modify this file if you have specialist needs.
+
+Since the runtime supports a
+[stateless system](https://clearlinux.org/features/stateless),
+it checks for this configuration file in multiple locations, two of which are
+built in to the runtime. The default location is
+`/usr/share/defaults/kata-containers/configuration.toml` for a standard
+system. However, if `/etc/kata-containers/configuration.toml` exists, this
+will take priority.
+
+The command below will list the full paths to the configuration files the runtime
```

Comment:
Lines 91-92: The command below lists the full paths to the configuration files that the runtime attempts to load. The first path that exists is used: